composer.json

```diff
@@ -11,12 +11,17 @@
         }
     ],
     "repositories": [
-        { "type": "composer", "url": "https://updates.ez.no/ttl" }
+        { "type": "composer", "url": "https://updates.ez.no/ttl" },
+        { "type": "vcs", "url": "https://github.com/adriendupuis/ezplatform-admin.git" }
     ],
+    "platform": {
+        "php": "7.3.14"
```

Comment:
Inside the Docker container, composer was “Killed” because of memory issue. My workstation have PHP 7.4.3 which differs from container. To execute Composer outside the container but ensure compatibility, I got to add this. I should loosen this constraint to `"php": "7.3"`.

I got memory limit issue also on my workstation so I decide to [download the .phar](https://getcomposer.org/download/) to be able to set unlimited memory:
`php -d memory_limit=-1 composer.phar require adriendupuis/ezplatform-admin dev-develop --no-scripts -vvv;`